articles/search_indexes.py

```diff
@@ -12,7 +12,7 @@ class TempArticleIndex(indexes.SearchIndex):
     def get_model(self):
         return Article
 
-    def index_queryset(self):
+    def index_queryset(self, **kwargs):
         return self.get_model().objects.active()
```

Comment:
`kwargs` is probably overkill, as [only `using` should ever be passed](https://github.com/django-haystack/django-haystack/blob/42f53cda9a770ff7daf2ff792cbcab5cd843e2a7/haystack/indexes.py#L125).